pkg/platform/kube/platform.go

```diff
@@ -758,6 +856,30 @@ func (p *Platform) UpdateFunctionEvent(updateFunctionEventOptions *platform.Upda
 		return errors.Wrap(err, "Failed to get a function event")
 	}
 
+	if functionName, found := functionEvent.Labels["nuclio.io/function-name"]; found {
```

Comment:
code duplicate
you may want to use `p.getFunction(...)` and maybe update its code to use `p.getter.Get(...)` as you used here. whatever works for you, just remove dup